### PR TITLE
[Hub Generated] Review request for Microsoft.Capacity to add version preview/2019-04-01

### PR DESCRIPTION
**Pull request by gautam714**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/gautam714/azure-rest-api-specs/dev-reservations-Microsoft.Capacity-2019-04-01_autorenew...master).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5876 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5876/commits/e476c255c0be78154ed2951fa280cd0ad13a6186

**Comment by gautam714** [comment]
Could somebody take a look that why are these 4 failing?
1. Preproduction (rest-api-specs)
2. Semantic Validator
3. continuous-integration/travis-ci/pr
4. public.rest-api-specs

Can you also add ARM people to review the APIs'?

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json [hunk 1]
```diff
@@ -1147,14 +1198,19 @@
...
+        "items": {
+            "type": "string",
+            "minItems": 1,
+            "maxItems": 1
+        }
+    },
+    "Renew": {
+        "type": "boolean",
+        "default": false,
+        "description":  "Setting this to true will automatically purchase a new resevation on the expiration date time."
```
Comment (nschonni) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]:
```suggestion
        "description":  "Setting this to true will automatically purchase a new reservation on the expiration date time."
```

**Comment by dsgouda** [comment]
@gautam714 Some of the examples are not valid jsons, please check and fix the errors reported [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/522334183)

**Comment by dsgouda** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5889 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5889/commits/4ad2c87e650ae3ffaf00264796564e27028ea3a5

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**